src/utils/AddToPlaylistBttn.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import {AddToPlaylistModal} from "./AddToPlaylistModal";
+
+export function AddToPlaylistBttn({ videoId }) {
+  const [playlistModal, setPlaylistModal] = useState("Hide");
+
+  function toggleModal() {
+    playlistModal === "Show"
+      ? setPlaylistModal("Hide")
+      : setPlaylistModal("Show");
+  }
+
+  return (
+    <>
+      {playlistModal === "Show" && (
+        <AddToPlaylistModal videoId={videoId} toggleModal={toggleModal} />
+      )}

```

Comment:
You should generally use boolean values if you're only doing 2 states, this prevents you from making typos in strings..

Reply:
yea agree